contrib/fwupd.spec.in

```diff
@@ -401,99 +402,99 @@ done
 %{_libdir}/girepository-1.0/FwupdPlugin-1.0.typelib
 /usr/lib/udev/rules.d/*.rules
 /usr/lib/systemd/system-shutdown/fwupd.shutdown
-%dir %{_libdir}/fwupd-plugins-3
```

Comment:
I guess we made a mistake at some point with this directory naming?

Reply:
Maybe; I think originally the soname of libfwupdplugin and and the plugin ABI were different, but realistically they're coupled and I think it makes sense to keep them the same. Maybe it makes sense to jump to **4** and make sure both change?

Reply:
> Maybe; I think originally the soname of libfwupdplugin and and the plugin ABI were different, but realistically they're coupled and I think it makes sense to keep them the same. Maybe it makes sense to jump to **4** and make sure both change?

Yes

Reply:
I've moved everything to 4 now.